E2E-ORS-Sample-For-Java/src/main/java/ObjectReplicationMonitor.java

```diff
@@ -0,0 +1,297 @@
+
+        String sourceContainerName = prop.getProperty("sourceContainerName");
+        String sourceStorageAccountConnectionString = prop.getProperty("sourceStorageAccountConnectionString");
+        String destinationContainerName = prop.getProperty("destinationContainerName");
+        String destinationStorageAccountConnectionString = prop.getProperty("destinationStorageAccountConnectionString");
+        String blobsToReplicatePrefix = prop.getProperty("blobsToReplicatePrefix");
+        String archiveMethod = prop.getProperty("archiveMethod");
+
+        // Creating a sample list of blobs to upload and replicate
+        String[] replicatedBlobList = new String[250];
+        int num = 0;
+        for (int i = 0; i < replicatedBlobList.length; i++) {
+
+            replicatedBlobList[i] = blobsToReplicatePrefix + num;
+            num++;
+
+        }
+
+        // Call method to setup blob in source container
+        setupSource(sourceStorageAccountConnectionString, sourceContainerName, replicatedBlobList);
+
+        // Call method to check the replication status of the blob in source container
+        System.out.println("\nOnce the replication process begins, the replication status will output...");
+        List<String> completedEventsList = checkReplication(sourceStorageAccountConnectionString, sourceContainerName, replicatedBlobList);
+
+        // Call method to check the contents of a replicated blob
+        checkContents(sourceStorageAccountConnectionString, sourceContainerName, destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+
+        // Call method to archive replicated blobs
+        if (archiveMethod.equals("batch")) {
+            archiveReplicatedDataWithBatch(destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+        }
+        else if (archiveMethod.equals("individual")) {
+            archiveReplicatedDataIndividually(destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+        }
+        else {
+            System.out.println("\nNo archive method selected");
+        }
+    }
+
+
+    private static void setupSource(String connectionString, String containerName, String[] blobList) {
+        /*
+        This method is to upload the blobs to the source container
+         */
+
+        System.out.println("\nUploading blobs to the source container...");
+
+        // Creating blobserviceclient in order to make containerclient
+        BlobServiceClient sourceBlobServiceClient = new BlobServiceClientBuilder().connectionString(connectionString).buildClient();
+
+        // Get the container client
+        BlobContainerClient sourceContainerClient = sourceBlobServiceClient.getBlobContainerClient(containerName);
+        if (!sourceContainerClient.exists()) {
+            sourceContainerClient.create();
+        }
+
+        // Iterate through list of blobs
+        for (String b : blobList) {
```

Comment:
Probably doesn't matter too much, but maybe more specific variable name? So like blobName instead of b